spec/features/edit_scheduled_messages_spec.rb

```diff
@@ -15,4 +15,24 @@
     expect(page).to have_content "Scheduled message updated successfully"
     expect(page).to have_content new_title
   end
+
+  scenario "from list of scheduled messages with tags" do
```

Comment:
I might change this to something like `scenario "edits tags for a scheduled message" do`